### PR TITLE
Partly fix #8581: add extra subdomains for cookie tests

### LINKED ISSUE #8581: Isolated domain for cookie tests?
We're looking into porting the IETF's cookie test suite (https://github.com/abarth/http-state/tree/master/tests) to WPT, which would be simplified if we could replicate its domain structure (https://github.com/abarth/http-state/blob/master/tests/data/hosts):

```
127.0.0.1 example.org
127.0.0.1 home.example.org
127.0.0.1 home.example.org.
127.0.0.1 sibling.example.org
127.0.0.1 subdomain.home.example.org
127.0.0.1 sibling.home.example.org
```

This basically boils down to two requests:

1.  Since y'all are looking into adding a new registrable domain in https://github.com/w3c/web-platform-tests/issues/2669, perhaps you could add a "just for cookie tests" domain? That would make the checks quite a bit simpler, as we wouldn't need to deal with noise from other cookies set by other tests on the same registrable domain.

2.  Can we add a subdomain of a subdomain (and another subdomain of the same subdomain) to the list of hosts?

### PR DESCRIPTION
**Pull request by gsnedders**
See #8581.

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/web-platform-tests/10953)

### CODE REVIEW

**Comment by gsnedders** [comment]
@jgraham pointed out yesterday that while @mikewest started off by saying in #8581 it "would be simplified if we could replicate its domain structure", the requests beneath it don't include using the exact same subdomains. And, of course, it'll need to be moved over to using `{{ host }}` at the very least, even if the subdomains are hardcoded, so it's not clear there's such a gain to exactly matching the hostnames?

**Review by mikewest — APPROVED** [review]
I assume these would be accessible via `{{hosts[][go.deeper]}}` and `{{hosts[alt][go.deeper]}}`? If so, LGTM!

**Comment by gsnedders** [comment]
@mikewest yes

File: tools/serve/serve.py [hunk 1]
```diff
@@ -663,7 +663,17 @@ def load_config(override_path=None, **kwargs):
                u"www1",
                u"www2",
                u"天気の良い日",
-               u"élève"}
+               u"élève",
+               u"deeper",
```
Comment (jgraham) [inline tools/serve/serve.py]: OK, o I"m going to sound like the most boring person, but I don't really like the whimsical names.

Is there something wrong with using the existing hosts and adding e.g. 
```
sub1a.www
sub1b.www
sub2a.sub1a.www
sub2b.sub1a.www
sub3a.sub2a.sub1a.www
```
and so on for whatever combinations you need?

**Review by jgraham — CHANGES_REQUESTED** [review]

File: tools/serve/serve.py (hunk 1 above)
Comment (gsnedders) [reply tools/serve/serve.py]: You're no fun!

Yours insincerely,

gsnedders.

**Comment by annevk** [comment]
The commit message should probably note that this fixes/closes #11219.

**Comment by gsnedders** [comment]
@jgraham Here's a less fun patch, following @annevk's suggestion from IRC to just repeat the existing subdomains.

**Review by jgraham — APPROVED** [review]

**Comment by jugglinmike** [comment]
For default configurations, this patch expands the number of distinct origins from 12 to 62 (actually 312, but [I think that's a bug](https://github.com/web-platform-tests/wpt/pull/12459)). If I'm reading the original feature request correctly, all that is technically required is "a subdomain of a subdomain (and another subdomain of the same subdomain)". We could satisfy this by introducing 2 new entries for a total of 14.  Elsewhere, @jgraham's wrote that "[it seems messy to add a lot of additional subdomains](https://github.com/web-platform-tests/wpt/issues/8581#issuecomment-388116372)."  @gsnedders I expect this might be a result of the feedback you got from Anne in IRC; could you say a bit about why you went with a combinatoric approach?

**Comment by annevk** [comment]
My suggestion was only about reusing existing names, if I remember correctly.

**Comment by gsnedders** [comment]
I have literally no memory why I went for so much repetition.

**Pull request merged**